NewHorizons/Builder/General/SectorBuilder.cs

```diff
@@ -22,7 +25,7 @@ public static Sector Make(GameObject planetBody, OWRigidbody owRigidBody, float
             sectorGO.AddComponent<OWTriggerVolume>();
 
             Sector S = sectorGO.AddComponent<Sector>();
-            S._name = (Sector.Name)24;
```

Comment:
I believe OWO was hardcoded to expect NH to use this value, so we'd have to coordinate this with Vesper. Maybe.

Reply:
I've checked the code thoroughly and there is no 24 anywhere.